src/main/java/services/petshop/steps/CommonMethodsPetSteps.java

```diff
@@ -1,4 +1,40 @@
 package services.petshop.steps;
 
+
+import io.restassured.response.Response;
+
+import org.junit.Assert;
+import net.thucydides.core.annotations.Step;
+import net.thucydides.core.annotations.Steps;
+import services.petshop.pojo.requests.Pet;
+
 public class CommonMethodsPetSteps {
+
+    @Steps
+    PostPetSteps postPetSteps;
+
+    @Steps
+    GetPetSteps getPetSteps;
+
+    @Step("When I retrieve the id of the created pet")
+    public void getTheIdOfTheCreatedPet(Pet mypet){
+        Response createPetR = postPetSteps.createNewPet(mypet);
+        Pet createdPet = createPetR.as(Pet.class);
+        System.out.println(getPetSteps.getPetWithId(createdPet.getId()));
+    }
+
+    @Step("Then I check the status code of the response when I perform an action successfully is 200")
```

Comment:
is a good approach to add a general method to check the status code, but I recommend to make it more general and send the code also as parameter.

Reply:
Also you have utils.methods.ReusableMethods#validateResponseStatusCode
ALready you have 2 methods which are doing the same thing. Just keep one.